net/devif/devif_loopback.c

```diff
@@ -123,13 +123,6 @@ int devif_loopback(FAR struct net_driver_s *dev)
         }
 
       NETDEV_TXDONE(dev);
-
-      /* Add the link layer header length for the next loop */
-
-      if (dev->d_len != 0)
-        {
-          dev->d_len += dev->d_llhdrlen;
```

Comment:
should we keep this?

Reply:
Add link layer length in loopback is unnecessary after below change checkin:
https://github.com/apache/nuttx/pull/7703
the length of the link layer will be increased in devif_out if any response from ipv4_input()